libmcount/dynamic.c

```diff
@@ -544,3 +637,71 @@ bool mcount_add_badsym(struct mcount_dynamic_info *mdi, unsigned long callsite,
 	list_add_tail(&badsym->list, &mdi->bad_syms);
 	return true;
 }
+
+void mcount_handle_dlopen(struct symtabs *symtabs, struct dl_phdr_info *info,
+			  char *mod_realpath, size_t size)
+{
+	struct uftrace_mmap *map;
+	struct mcount_dynamic_info *mdi;
+	unsigned long vaddr = 0, text_addr = 0;
+	unsigned int text_size = 0, i;
+	bool text_section_found = false;
+
+	char* modname = basename(mod_realpath);
+
+	if (!find_module_list(modname))
+		return;
+
+	char *libname = xstrdup(modname);
+	int namelen = ALIGN(strlen(libname) + 1, 4);
```

Comment:
move these declarations with other declarations.

Reply:
was reflected what you said.